test/dlc_tests/ec2/pytorch/training/test_pytorch_training.py

```diff
@@ -569,15 +547,14 @@ def test_pytorch_amp_inductor(
     execute_ec2_training_test(ec2_connection, pytorch_training, PT_AMP_INDUCTOR_CMD, timeout=1500)
 
 
-@pytest.mark.skip_pt21_test
-@pytest.mark.skip_pt22_test
+@pytest.mark.skip_serialized_release_pt_test
+@pytest.mark.skip_s3plugin_test
```

Comment:
no in-support image is using this anymore according to the fixture. I think it is safe to just remove the two s3 plugin tests here

Reply:
I just did a search throughout the repo on s3plugin test. Looks like they're not used but there are a lot of deletion to be done across ec2 and sm tests. I'd rather decouple this from currency and do a clean up of this test separately.